### PR TITLE
specify OPENAI_API_BASE

### PR DESCRIPTION
**Pull request by lem0ke**
In "transcribe/app/transcribe/gpt_responder.py" only modified 141 lines and 231 lines of code, 251 lines of code I did not understand the code writing, in order to prevent the tricky so did not act rashly. In "transcribe/sdk/transcriber_models.py", 194 lines were modified by adding ", base_url=base_url". I'm a very poor coder, so I haven't made too many changes and only partially followed the guidelines you wrote out for me, so please forgive me if there are any mistakes. If there are more changes that need to be made, please help me to continue with the process, thank you very much! 🙏

### CODE REVIEW

File: app/transcribe/gpt_responder.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def generate_response_from_transcript_no_check(self) -> str:
     def create_client(self, api_key: str):
```
Comment (mang0sw33t) [inline app/transcribe/gpt_responder.py:138]: Add the parameter base_url here as well.
So change the line 
`create_client(self, api_key: str)`
to 
`create_client(self, api_key: str, base_url: str = None)`

File: app/transcribe/gpt_responder.py [hunk 2]
```diff
@@ -228,7 +228,7 @@ def __init__(self,
         root_logger.info(OpenAIResponder.__name__)
```
Comment (mang0sw33t) [inline app/transcribe/gpt_responder.py:228]: Change the lines

```
            def __init__(self,
                 config: dict,
                 convo: conversation.Conversation,
                 save_to_file: bool = False,
                 response_file_name: str = 'logs/response.txt')
```
to

```
            def __init__(self,
                 config: dict,
                 convo: conversation.Conversation,
                 save_to_file: bool = False,
                 base_url: str = None,
                 response_file_name: str = 'logs/response.txt')
```

**Review by mang0sw33t — CHANGES_REQUESTED** [review]
After making the above requested changes, execute the program and make sure it executes correctly.

The ideal solution is to make the parameter configurable in parameter.yaml, overrride.yaml.

Those changes would be hard to explain as comments in PR review. We need to talk about that live 
or I will make those changes once this PR is merged
or you can give me access to push code to this repo and I can make those changes as part of this PR.

**Comment by lem0ke** [comment]
> After making the above requested changes, execute the program and make sure it executes correctly.
> 
> The ideal solution is to make the parameter configurable in parameter.yaml, overrride.yaml.
> 
> Those changes would be hard to explain as comments in PR review. We need to talk about that live or I will make those changes once this PR is merged or you can give me access to push code to this repo and I can make those changes as part of this PR.

I've granted you access in the project I'm forking, thanks again for doing this!

**Comment by vivekuppal** [comment]
> I've granted you access in the project I'm forking, thanks again for doing this!
I will do this. 
@lem0ke I am doing this for you and @vivekuppal
Make sure he gets what he asks of you when he asks for it.

**Comment by mang0sw33t** [comment]
> I've granted you access in the project I'm forking, thanks again for doing this!

I will do this sometime today.
@lem0ke I am doing this for you and @vivekuppal.
Make sure he gets what he asks of you, when he asks for it.

**Comment by lem0ke** [comment]
> > I've granted you access in the project I'm forking, thanks again for doing this!
> 
> I will do this sometime today. @lem0ke I am doing this for you and @vivekuppal. Make sure he gets what he asks of you, when he asks for it.

For the project I forked, I have granted @vivekuppal permissions along with yours. If more help is needed, I'll do my best to provide it. Replies may not be timely due to time zones, it's already midnight here, but I'll try to reply to messages promptly!

**Comment by mang0sw33t** [comment]
We may not have asks today, but in the near future. Happy Transcribing!

**Review by mang0sw33t — APPROVED** [review]

**Pull request merged**